src/main/resources/application.yaml

```diff
@@ -49,5 +56,13 @@ oidc:
 ccd:
   data-store:
     host: ${CCD_DATA_STORE_HOST:http://localhost:4452}
+    whitelisted-urls: /ccd/searcCases\?ctid.*
```

Comment:
what happens if a user changes the order of the query params? for example:
searchCases/use_case="aaaa"&ctid="bbbb"

Reply:
I wouldn't include query params at all

Reply:
It will fail. This is a regular regex. Will change to 'whitelisted-urls: /ccd/searchCases.*' So all parameters are allowed in any order. Have added ctid to test it will work with parameters and noticed 'h' letter is missing, as I have been breaking it for testing pourpose.